chatterbot/adapters/logic/closest_match.py

```diff
@@ -4,6 +4,12 @@
 
 
 class ClosestMatchAdapter(BaseMatchAdapter):
+    """
+    The ClosestMatchAdapter extends the BaseMatchAdapter to find
+    the closest matching response. It creates a response by
+    using fuzzywuzzy's process class to extract the most similar
+    response to the input.
```

Comment:
Maybe just leave out the "The ClosestMatchAdapter extends the BaseMatchAdapter to find the closest matching response."?

Also, instead of implying that fuzzywuzzy just magically chooses a response, perhaps it would be more helpful to say something like
> This adapter selects a response to an input statement by selecting the closest known matching statement based on the Levenshtein Distance between the text of each statement.